examples/notebooks/show_remote_demo_cubes.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
It is a cool notebook (I should say that as we use this in ESDL). However, this is more an example of using S3 
not of xcube. I note later that this could be integral part of xcube. Then it would become an xcube example. 

You could open a feature reqeust for this.

Reply:
@forman What do you think about Helges suggestion?